ui/app/AppLayouts/stores/RootStore.qml

```diff
@@ -38,10 +38,6 @@ QtObject {
     }
 
     property string communityKeyToImport
-    onCommunityKeyToImportChanged: {
```

Comment:
@jrainville this should give some decent speedups too :)

We were basically calling the `prepareTokenModelForCommunity` twice needlessly